src/Fixer/Phpdoc/PhpdocIndentFixer.php

```diff
@@ -105,7 +105,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
 
             $newPrevContent = $this->fixWhitespaceBeforeDocblock($prevToken->getContent(), $indent);
             if ($newPrevContent) {
-                $tokens[$prevIndex] = new Token(array($prevToken->getId(), $newPrevContent));
+                $tokens[$prevIndex]->setContent($newPrevContent);
```

Comment:
not 100% sure about this fix because I don't know why a new Token was created here

Reply:
hmmm... fair enough :)
do you have a suggestion how to proceed here? I'm not sure what would be best

Reply:
I think you have to check whether `$prevToken->getId()` is null and only pass the content as parameter of `Token` constructor in such case instead of an array (id + content).